src/Routes/ImageManager/Images.js

```diff
@@ -81,6 +100,106 @@ const Images = () => {
     }),
     shallowEqual
   );
+
+  useEffect(() => {
+    if (data) {
+      setRows(
+        flatten(
+          data.map(item => {
+            const statusIconHash = {
+              "Ready": <CheckCircleIcon color="var(--pf-global--success-color--100)" />,
+              "Image build in progress": <InProgressIcon color="var(--pf-global--palette--blue-400)" />,
```

Comment:
For now it is fine but in the future we will have to use https://github.com/patternfly/patternfly-react/blob/master/packages/react-tokens/README.md instead of "var(...)"

Reply:
As mentioned above, I'd personally add multiple classes with PF CSS vars. The react-tokens has some problems with minification so I would like to avoid them and use CSS vars in scss file.

Reply:
@karelhala sounds good to me! @matoval - create classes in App.scss that uses this color. Although look what I did in the icon status mapper and see if what I did is simpler and works.

Reply:
Can I use [StatusLabel](https://github.com/RedHatInsights/edge-frontend/blob/master/src/Routes/ImageManagerDetail/StatusLabel.js) to get the icon and text here?